test/e2e/scalability/timer/timer.go

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
I think this pkg is generic enough to be usable outside of scalability. IMO you should place this in `test/e2e/framework`